### PR TITLE
Ledger

### PR DESCRIPTION
**Pull request by theautobot**

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#83](https://github.com/zold-io/zold.github.io/issues/83) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#83](https://github.com/zold-io/zold.github.io/issues/83); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: html/ledger.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>/ledger</title>
+    <meta name="description" content="Stress test of the entire Zold network"/>
```
Comment (yegor256) [inline html/ledger.html]: @theautobot it's not stress test, right?

File: html/ledger.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <link rel="shortcut icon" href="https://www.zold.io/logo.png"/>
+    <link href="https://cdn.rawgit.com/yegor256/tacit/gh-pages/tacit-css-1.3.0.min.css" rel="stylesheet"/>
+    <link href="@@VERSION/css/health.min.css" rel="stylesheet"/>
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js">
+      // Should be empty
+    </script>
+    <script src="https://cdn.rawgit.com/yegor256/colorizejs/gh-pages/colorizejs.min.js">
+      // Should be empty
+    </script>
+    <script src="@@VERSION/js/stress.js">
```
Comment (yegor256) [inline html/ledger.html]: @theautobot typo?

File: html/ledger.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      // Should be empty
+    </script>
+  </head>
+  <body onload="stress_init();">
```
Comment (yegor256) [inline html/ledger.html]: @theautobot typo?

**Comment by yegor256** [comment]
@theautobot remove `.idea` files from the branch, please

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor release, tag is `0.9.0`

**Comment by 0crat** [comment]
Job `gh:zold-io/zold.github.io#83` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#83](https://github.com/zold-io/zold.github.io/issues/83) is now out of scope

**Pull request merged**